cmd/flux/bootstrap_git.go

```diff
@@ -199,6 +200,15 @@ func bootstrapGitCmdRun(cmd *cobra.Command, args []string) error {
 		RecurseSubmodules: bootstrapArgs.recurseSubmodules,
 	}
 
+	var caBundle []byte
+	if bootstrapArgs.caFile != "" {
+		var err error
+		caBundle, err = ioutil.ReadFile(bootstrapArgs.caFile)
```

Comment:
`ioutil` is deprecated since Go 1.16. From the top of my head the successor is `os#ReadFile` (or something something).

Reply:
Oooh, you're right. Will fix.

Reply:
done